docs/modules/ROOT/pages/integrations/database.adoc

```diff
@@ -0,0 +1,114 @@
+= Database Integrations
+
+== Overview
+
+KubeArchive requires a database to store resources.
+This document list the databases supported, how to set up the database schema
+and how to add new database integrations.
+
+=== PostgreSQL
+
+The only currently full supported.
```

Comment:
```suggestion
Currently PostgreSQL is the only fully supported.
```

Reply:
Applied the changes with a little modification:

`Currently, PostgreSQL is the only fully supported database engine.`

WDYT?